packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -608,14 +616,11 @@ export function useCallback<T>(
   return callback;
 }
 
-// a variable to store current component while we compare memo values
-let cachedCurrentlyRenderingFiber = null;
-
 export function useMemo<T>(
   nextCreate: () => T,
   inputs: Array<mixed> | void | null,
 ): T {
-  cachedCurrentlyRenderingFiber = currentlyRenderingFiber = resolveCurrentlyRenderingFiber();
+  let cachedCurrentlyRenderingFiber = (currentlyRenderingFiber = resolveCurrentlyRenderingFiber());
```

Comment:
Nit: Maybe just "fiber" everywhere? It's local so doesn't need too much introducing.

Reply:
didn't want to encourage reading from it, but I guess that's actually fine